client-display/src/userSkills/skill/SelfReportSkillModal.vue

```diff
@@ -40,7 +40,7 @@ limitations under the License.
            aria-describedby="reportSkillMsg"
            :aria-label="isJustitificationRequired ? 'Optional request approval message' : 'Required request approval message'"
            class="form-control" :placeholder="`Message (${isJustitificationRequired ? 'required' : 'optional'})`"/>
-    <div v-if="isApprovalRequired" :class="{ 'float-right':true, 'text-small': true, 'text-danger': charactersRemaining < 0 }" data-cy="charactersRemaining">{{charactersRemaining}} characters remaining <i v-if="charactersRemaining < 0" class="fas fa-exclamation-circle"/></div>
+    <div v-if="isApprovalRequired" :class="{ 'float-right':true, 'text-small': true, 'text-danger': charactersRemaining < 0 }" data-cy="charactersRemaining">{{formattedRemaining}} characters remaining <i v-if="charactersRemaining < 0" class="fas fa-exclamation-circle"/></div>
```

Comment:
instead of new util suggesting to use the existing number filter, like this {{ charactersRemaining | number }}

Reply:
Ha, that would indeed make dramatically more sense!